serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc

```diff
@@ -1 +1,700 @@
+quarkus.oidc-client.acme_financial_oauth.credentials.client-secret.value=secret <8>
+----
+
+<1> Is the package name for the automatically generated classes that implements the access to the all operations defined in the file `acme-financial-service.yml`.
+<2> Is the root URL to access all the operations defined in the file `acme-financial-service.yml`. For the `exchangeRate` operation, an absolute URL such as \http://localhost:8483/financial-service/exchange-rate is automatically generated.
+
+<3> Disable the Oauth2 server endpoints discovery since the endpoints provided in the <<acme-financial-service-yml, acme-financial-service.yml>> file will be used instead.
+<4> Is the authentication URL of the Oauth2 server.
+<5> Is the relative path or absolute URL of the Oauth2 token endpoint which issues access and refresh tokens.
+<6> Is the client-id to identify the Serverless Workflow against the authorization service, such as `kogito-app`. This identifier must be provided by Acme.
+<7> Is the method to use at the time to send the client-secret for the authentications when the client grant type is used.
+<8> Is the client secret to authenticate the Serverless Workflow against the authorization service when the client grant type is used. This secret must be provided by Acme.
+
+To configure the `acme_financial_service_oauth` {quarkus_guide_security_openid_connect_client_url}[Quarkus OpenId Connect Client (OIDC)] you must follow the rules described in {authentication-support-for-openapi-services-oauth-example}[Example of OAuth2 authentication].
+And the particular attributes depends on the Oauth2 server and authorization flow to use, you can get this information from the OpenAPI specification of the target service <<acme-financial-service-yml, acme-financial-service.yml>>.
+
+[NOTE]
+====
+You can use the alternatives defined in the {quarkus_guide_config_reference_url}[Quarkus configuration reference guide] to configure these properties.
+A common usage is to define {quarkus_guide_config_reference_environment_variables_url}[environment variables] to set the authentication secrets, and you can also use the {quarkus_guide_credentials_provider_url}[Quarkus Credentials Provider] framework.
+====
+
+== Running the example
+
+To run the example you must follow these steps:
+
+=== Get the example
+
+. In a new terminal, git clone the kogito-examples repository and navigate to this directory:
+
+[source, bash]
+----
+$ git clone https://github.com/kiegroup/kogito-examples.git
+
+$ cd kogito-examples/serverless-workflow-examples/serverless-workflow-oauth2-orchestration-quarkus
+----
+
+=== Build the example
+
+[start=2]
+. Build the example using Maven:
+
+[source, bash]
+----
+$ mvn clean install
+----
+
+=== Start Keycloak
+
+[start=3]
+. In a new terminal, start the Keycloak server:
+
+[source, bash]
+----
+$ cd kogito-examples/serverless-workflow-examples/serverless-workflow-oauth2-orchestration-quarkus/scripts
+
+$ cd ./startKeycloak.sh
```

Comment:
maybe put these in different code snippet, since users might want to copy and past those.